db/tasks.php

```diff
@@ -28,10 +28,10 @@
     array(
         'classname' => '\logstore_xapi\task\emit_task',
         'blocking' => 0,
-        'minute' => '*/5',
+        'minute' => '*/1',
```

Comment:
sure, it was just added in my code because I can't wait 5 minutes to debug every time. Do you think that could be an interesting improvement moving that in the settings?